TAScheduler/AcceptanceTests/AssignTAtoLab tests.py

```diff
@@ -0,0 +1,120 @@
+import datetime
+
+from django.test import TestCase
+
+from TAScheduler.models import User, TA, Course, TAToCourse
+
+
+#
+class SuccessfulAssignment(TestCase):
+    user = None
+    ta = None
+    course = None
+    tatocourse = None
+
+    def setUp(self):
+        self.user = User.objects.create(email_address="ta@class.com", password="password", first_name="first",
+                                        last_name="last", home_address="there", phone_number=1234567890)
+
+        self.ta = TA.objects.create(user=self.user)
+
+        self.course = Course.objects.create(course_id=100, semester="fall 2023", name="test",
+                                            description="test", num_of_sections=2, modality="online", credits=1)
+
+        self.tatocourse = TAToCourse.objects.create(ta=self.ta, course=self.course)
+
+    def test_creation(self):
+        temp = TAToCourse.objects.create(ta=self.ta, course=self.course)
+        self.assertIsNotNone(temp, "TAToCourse does not make anything")
+        self.assertEqual(temp.course, self.course, "TAToCourse does not make correct course")
+        self.assertEqual(temp.ta, self.ta, "TAToCourse does not make correct TA")
+
+
```

Comment:
Add a test for "confirmation message" i.e.: resp.context["message"] = "SUCCESSFULLY CREATED NEW PASSWORD"